src/Sylius/Bundle/CoreBundle/Migrations/Version20230327121633.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230327121633 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Allow longer access and refresh tokens for OAuth users';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE sylius_user_oauth CHANGE access_token access_token VARCHAR(8192) DEFAULT NULL, CHANGE refresh_token refresh_token VARCHAR(8192) DEFAULT NULL');
```

Comment:
For your information, 8192 will not work with MariaDB and innoDB. https://mariadb.com/kb/en/troubleshooting-row-size-too-large-errors-with-innodb/

Reply:
Think this has been already addressed in #14930, the update changed the field type from varchar simply to text.